addons/calendar/views/calendar_views.xml

```diff
@@ -75,7 +71,7 @@
         <field name="name">calendar.event.tree</field>
         <field name="model">calendar.event</field>
         <field name="arch" type="xml">
-            <tree string="Meetings" sample="1" multi_edit="1">
+            <tree string="Meetings" sample="1" multi_edit="1" default_order="start asc">
```

Comment:
Have to say this is strange ... why seeing your meetings from 2012 first ?

Reply:
Indeed, we may need a default filter with start date today or start of this week. I'll update this.

But as it is, no order (creation order?) is ~~anarchy~~ unreadable.

Reply:
Removing this here, will be further discussed later.